theme/styles/parts/defaultForm.sass

```diff
@@ -0,0 +1,2 @@
+form:not([class])
+	@extend %form
```

Comment:
Název souboru neodpovídá použité (žádné) třídě, ale html elementu. Změnit prosím na `.defaultForm`/`.form`.